### DISCUSSION TITLE
What TTS model from coqui-ai is used for this website?

### DISCUSSION THREAD

**Discussion by Vagif12** [coqui-ai/TTS#1290] [General Q&A]
Just came across:
https://audioread.com/

And the Experimental TTS Engine sounds extremely human-like. 
I dug around and can't seem to find the origin of this model.
Does anybody know?
I'm assuming it's a voice from the VCTK Dataset(https://www.tensorflow.org/datasets/catalog/vctk) but I couldn't find the exact one.
Any thoughts?
Thanks!

**Comment by erogol** [comment]
They might be using something propriatery.
